docs/quickstart/http-transport.md

```
@@ -69,6 +69,11 @@ docker run -p 8080:8080 \
 npx @maximhq/bifrost -port 8080
 ```
 
+> **🔄 Smart Configuration Loading**: Bifrost intelligently manages configuration sources:
+> - **If `config.json` exists**: Checks if the file has changed. If unchanged, loads from database (fast path). If changed, uses file as source of truth and syncs to database.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)